src/main/java/io/github/thebusybiscuit/slimefun4/implementation/listeners/BlockListener.java

```diff
@@ -141,15 +141,15 @@ public void onBlockBreak(BlockBreakEvent e) {
             }
         }
 
-        if (!e.isCancelled()) {
-            checkForSensitiveBlockAbove(e.getPlayer(), e.getBlock(), item);
+        int fortune = getBonusDropsWithFortune(item, e.getBlock());
```

Comment:
Can we move fortune down to just above callToolHandler?
There's no reason for it to be this high.

Reply:
I mean it does have to be right? It's used in the next method call, it looks like it needs to be dropped into that if block but otherwise it's in the right place, at least that's how it looks anyways

Reply:
Done.